include/knowhere/comp/index_param.h

```diff
@@ -81,6 +81,35 @@ constexpr const char* M = "m";          // PQ param for IVFPQ
 constexpr const char* SSIZE = "ssize";
 constexpr const char* REORDER_K = "reorder_k";
 constexpr const char* WITH_RAW_DATA = "with_raw_data";
+// RAFT Params
+constexpr const char* REFINE_RATIO = "refine_ratio";
```

Comment:
should these parameters be names as something like `raft_refine_ratio` instead of `refine_ratio` ? I'm not sure though

Reply:
good suggest, I will push another pr do this.